assets/js/modules/analytics-4/datastore/properties.test.js

```diff
@@ -209,6 +209,46 @@ describe( 'modules/analytics-4 properties', () => {
 					.dispatch( MODULES_ANALYTICS_4 )
 					.selectProperty( propertyID );
 
+				expect(
+					registry.select( MODULES_ANALYTICS_4 ).getPropertyID()
+				).toBe( propertyID );
+				expect(
+					registry.select( MODULES_ANALYTICS_4 ).getWebDataStreamID()
+				).toBe( fixtures.webDataStreams[ 0 ]._id );
+				expect(
+					registry.select( MODULES_ANALYTICS_4 ).getMeasurementID()
+				).toBe(
+					// eslint-disable-next-line sitekit/acronym-case
+					fixtures.webDataStreams[ 0 ].webStreamData.measurementId
+				);
+			} );
+
+			it( 'should set property ID and reset datastream with measurement IDs when no web data streams are available', async () => {
```

Comment:
I realise this wording is not introduced here but the word "with" is confusing here as it suggests that measurement IDs are somehow being used to reset the datastream...

A simply change to "and" should clear this up, I'll make the change myself.

```suggestion
			it( 'should set property ID, and reset datastream and measurement IDs when no web data streams are available', async () => {
```